src/pages/search/advanced_search/LanguageFilter.tsx

```diff
@@ -0,0 +1,79 @@
+import { Box, Chip, FormControl, InputLabel, MenuItem, Select } from '@mui/material';
+import { useTranslation } from 'react-i18next';
+import { useHistory } from 'react-router';
+import { ResultParam } from '../../../api/searchApi';
+import i18n from '../../../translations/i18n';
+import { dataTestId } from '../../../utils/dataTestIds';
+import { languageOptions } from '../../registration/DescriptionPanel';
+
+export const LanguageFilter = () => {
+  const { t } = useTranslation();
+  const history = useHistory();
+  const searchParams = new URLSearchParams(history.location.search);
+  const languageParam = searchParams.get(ResultParam.PublicationLanguageShould)?.split(',') || [];
+
+  const handleChange = (selectedUris: string[] | null) => {
+    if (selectedUris && selectedUris.length > 0) {
+      const languages = selectedUris
+        .map((iso6393Code) => {
+          const selectedLanguage = languageOptions.find((language) => language.iso6393Code === iso6393Code);
+          return selectedLanguage ? selectedLanguage.iso6393Code : null;
+        })
+        .filter(Boolean);
+
+      if (languages.length > 0) {
+        searchParams.set(ResultParam.PublicationLanguageShould, languages.join(','));
+      } else {
+        searchParams.delete(ResultParam.PublicationLanguageShould);
+      }
+    } else {
+      searchParams.delete(ResultParam.PublicationLanguageShould);
+    }
+
+    history.push({ search: searchParams.toString() });
+  };
+
+  const handleDelete = (iso6393CodeToDelete?: string) => () => {
+    selectedLanguages.length > 1
+      ? handleChange(languageParam.filter((iso6393Code) => iso6393Code !== iso6393CodeToDelete))
+      : handleChange([]);
+  };
+
+  const selectedLanguages = languageParam
+    .map((iso6393Code) => languageOptions.find((language) => language.iso6393Code === iso6393Code))
+    .filter(Boolean);
+
+  return (
+    <Box sx={{ display: 'flex', flexDirection: 'column', gap: '0.5rem', maxWidth: '15rem' }}>
+      <FormControl fullWidth sx={{ width: '15rem' }}>
```

Comment:
Vanskelig å se hva som bestemmer bredden nå når det er oppgitt `15rem` to steder. Dette virker likt?
```suggestion
    <Box sx={{ display: 'flex', flexDirection: 'column', gap: '0.5rem', width: '15rem' }}>
      <FormControl fullWidth>
```

Reply:
ja, ble litt knot her